common/dconfig/configmgr.go

```diff
@@ -339,46 +369,75 @@ func (cfgMgr *CassandraConfigManager) mkKVTreeForSvc(service string, items []*m.
 			continue
 		}
 
-		versions[version] = struct{}{}
+		if sku != wildcardToken && host != wildcardToken {
+			key := strings.Join([]string{service,version, sku, host,cfgKey}, ".")
+			sku = wildcardToken
+			items[i].Sku = common.StringPtr(wildcardToken)
+			cfgMgr.logger.WithField(key, cfgValue).Warn("Forcing sku=*;host specific items MUST be sku agnostic")
```

Comment:
It would be good to keep the log-field "tag" to be a constant string literal and move the variable/dynamic part to the "value" of the tag.